tensorboard/pip_package/requirements.txt

```diff
@@ -28,7 +28,7 @@ numpy >= 1.12.0
 # See: http://b/182876485
 # See: https://github.com/protocolbuffers/protobuf/issues/9954#issuecomment-1128283911
 # See: https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/tools/pip_package/setup.py?q=protobuf
-protobuf >= 3.9.2, < 4
```

Comment:
I believe we need just
`protobuf >= 3.9.2` here, TF hasn't migrated to protobuf 4 yet but needs tb to accept protobuf 4 first to facilitate TF migration process